taskvine/src/manager/vine_manager.c

```diff
@@ -405,7 +405,11 @@ static vine_msg_code_t vine_manager_recv_no_retry(struct vine_manager *q, struct
 		result = handle_cache_update(q, w, line);
 	} else if (string_prefix_is(line, "cache-invalid")) {
 		result = handle_cache_invalid(q, w, line);
-	} else if (string_prefix_is(line, "transfer-address")) {
+	} 
```

Comment:
This seems incomplete.  Did you mean to finish this, or is it meant to be part of a future PR?